tiny_cnn/core/kernels/nnp_deconv2d_kernel.h

```diff
@@ -35,21 +35,21 @@ namespace tiny_cnn {
 namespace core {
 namespace kernels {
 
-nnp_deconvolution_algorithm nnp_algorithm() {
```

Comment:
@bhack Yeah I noticed this just when @edgarriba informed that NNPack just support for typical convolution, I have another method to implement deconvolution using convolution. That will make tiny_cnn able to run deconv based on NNPack.

The method detail is roughly about:
1. padding input with zeros of **2(weight.width-1)** rather than **weight.width-1** in convolution with padding::same strategy.
2. reverse the convolution kernel in 2D space and remain the channel sequence.